libraries/chain/controller.cpp

```diff
@@ -295,9 +295,9 @@ struct controller_impl {
     blog( cfg.blocks_dir, cfg.prune_config ),
     fork_db( cfg.blocks_dir / config::reversible_blocks_dir_name ),
     wasmif( cfg.wasm_runtime, cfg.eosvmoc_tierup, db, cfg.state_dir, cfg.eosvmoc_config, !cfg.profile_accounts.empty() ),
-    resource_limits( db, [&s]() { return s.get_deep_mind_logger(); }),
+    resource_limits( db, [&s]() { return s.get_deep_mind_logger(false); }),
     authorization( s, db ),
-    protocol_features( std::move(pfs), [&s]() { return s.get_deep_mind_logger(); } ),
+    protocol_features( std::move(pfs), [&s]() { return s.get_deep_mind_logger(false); } ),
```

Comment:
Seems like this should be a lambda that take `bool trx_is_transient` and pass that on to `controller.get_deep_mind_logger()`.